packages/editor/src/client/routes/commentList.tsx

```diff
@@ -0,0 +1,408 @@
+                />
+                <IconButton
+                  icon={<Icon icon="close" />}
+                  color="red"
+                  circle
+                  size="sm"
+                  style={{marginLeft: '5px'}}
+                  onClick={() => {
+                    setCurrentComment(rowData)
+                    setConfirmAction(ConfirmAction.Reject)
+                    setConfirmationDialogOpen(true)
+                  }}
+                />
+              </>
+            )}
+          </Cell>
+        </Column>
+      </Table>
+
+      <Pagination
+        style={{height: '50px'}}
+        lengthMenu={DEFAULT_TABLE_PAGE_SIZES}
+        activePage={page}
+        displayLength={limit}
+        total={data?.comments.totalCount}
+        onChangePage={page => setPage(page)}
+        onChangeLength={limit => setLimit(limit)}
+      />
+      <Modal
+        show={isConfirmationDialogOpen}
+        width="sm"
+        overflow
+        onHide={() => setConfirmationDialogOpen(false)}>
+        <Modal.Header>
+          <Modal.Title>
+            <div>{t(mapModalTitle())}</div>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <DescriptionList>
+            <DescriptionListItem label={t('comments.panels.userName')}>
+              {currentComment?.user.name || t('comments.panels.untitled')}
+            </DescriptionListItem>
+            <DescriptionListItem label={t('comments.panels.createdAt')}>
+              {currentComment?.createdAt && new Date(currentComment.createdAt).toDateString()}
+            </DescriptionListItem>
+            <DescriptionListItem label={t('comments.panels.updatedAt')}>
+              {currentComment?.modifiedAt && new Date(currentComment.modifiedAt).toDateString()}
+            </DescriptionListItem>
+            <DescriptionListItem label={t('comments.panels.revisions')}>
+              <Timeline align="left">
+                {currentComment?.revisions?.length
+                  ? currentComment?.revisions?.map(({text, createdAt}, i) => (
+                      <Timeline.Item key={i}>
+                        <div>{new Date(createdAt).toLocaleString()}</div>
+                        <RichTextBlock
+                          disabled
+                          // TODO: remove this
+                          onChange={console.log}
+                          value={text}
+                        />
+                      </Timeline.Item>
+                    ))
+                  : null}
+              </Timeline>
+            </DescriptionListItem>
+            {confirmAction === ConfirmAction.Reject ||
+            confirmAction === ConfirmAction.RequestChanges ? (
+              <DescriptionListItem
+                label={t(
+                  confirmAction === ConfirmAction.Reject
+                    ? 'comments.panels.rejectionReason'
+                    : 'comments.panels.requestChangesReason'
+                )}>
+                <Dropdown
```

Comment:
Selected reasons are not shown
![deepin-screen-recorder_Select area_20210208100647](https://user-images.githubusercontent.com/2340416/107198616-9903d000-69f5-11eb-89e0-a645fca0d1cc.gif)